examples/formik/js/async-validator/index.js

```diff
@@ -0,0 +1,84 @@
+// Helper styles for demo
+import './helper.css';
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
```

Comment:
Why is it all html inside an `h1` element?

Reply:
Is from Formik Examples, going to add some meaningful text